packages/shared-components/src/components/onto-tooltip/onto-tooltip.tsx

```diff
@@ -0,0 +1,103 @@
+import tippy, {Instance, Placement} from 'tippy.js';
+
+@Component({
+  tag: 'onto-tooltip',
+  styleUrl: 'onto-tooltip.scss',
+  shadow: false,
+})
+export class OntoTooltip {
+  private tooltip: Instance;
+  private showFunction;
+  private hideFunction;
+
+  @Element() el: HTMLElement;
+
+  /**
+   * The content ot tooltip that will be displayed when mouse is over the element.
+   */
+  @Prop() tooltipContent: string;
+
+  /**
+   * Specifies the position of the tooltip.
+   */
+  @Prop() placement: Placement;
+
+  @Watch('tooltipContent')
+  configurationChanged() {
+    this.update();
+  }
+
+  componentDidLoad() {
+    this.init();
+  }
+
+  disconnectedCallback() {
+    this.destroy();
+  }
+
+  render() {
+    return (
+      <Host class="yasgui-tooltip">
+        <slot />
+      </Host>
+    );
+  }
+
+  private update() {
+    this.init();
+    this.tooltip.setContent(this.tooltipContent);
+  }
+
+  private init() {
+    if (this.tooltip) {
+      return;
+    }
+    const options = {
+      content: this.tooltipContent,
+      trigger: 'manual',
+      placement: this.placement as Placement,
+      allowHTML: true,
+      triggerTarget: this.el,
+      theme: "onto-tooltip",
+      /**
+       * The tippy library has some conflict with the Google Chart Editor. The editor adds a div element with the "jfk-tooltip" class.
+       * When the mouse hovers over a chart editor element, this div is positioned accordingly.
+       * When the mouse leaves the element, a "jfk-tooltip-hidden" class is added, and the div tag is hidden.
+       * For some reason, when a 'onto-tooltip' is open, the "jfk-tooltip-hidden" class is removed, and the Google Chart tooltip is displayed along with
+       * 'onto-tooltip' (maybe Google uses an old version of tippy or popover and there is a conflict in implementations).
+       * When 'onto-tooltip' is open, we add the class 'hidden' to the Google Chart tooltip to hide it.
+       */
+      onShow: () => document.querySelectorAll('.jfk-tooltip').forEach(popper => popper.classList.add('hidden')),
+      /**
+       * When 'onto-tooltip' is closed, we remove the 'hidden' class to allow the Google Chart tooltip to work properly.
+       */
+      onHide: () => document.querySelectorAll('.jfk-tooltip').forEach(popper => popper.classList.remove('hidden'))
+    };
+    this.tooltip = tippy(this.el, options);
+
+    this.showFunction = this.createShowFunction(this.tooltip);
+    this.hideFunction = this.createHideFunction(this.tooltip);
+
+    this.el.addEventListener('mouseover', this.showFunction);
+    this.el.addEventListener('mouseleave', this.hideFunction);
+  }
+
+  private createShowFunction(tooltip: Instance) {
+    return () => {
+      if (this.tooltipContent) {
+        tooltip.show();
+      }
+    }
+  }
+
+  private createHideFunction(tooltip: Instance) {
+    return () => tooltip.hide();
+  }
+
+  private destroy() {
+    this.el.removeEventListener('mouseover', this.showFunction);
```

Comment:
Are you sure these really unsubscribe the event handlers? Given how you create them I'm not quite convinced it really works.

Reply:
The functions are created at runtime, and the registration and unregistration of the listener use the same function reference. I added some console logging, and it looks like it is working as expected.